charts/camunda-platform/templates/optimize/deployment.yaml

```diff
@@ -191,6 +273,12 @@ spec:
           emptyDir: {}
         - name: camunda
           emptyDir: {}
+        {{- if or .Values.global.elasticsearch.tls.existingSecret .Values.global.opensearch.tls.existingSecret }}
```

Comment:
This isn't old. 
I created this if statement since I need to add the body of the if statement into the template file if the user provides a tls secret for either elasticsearch or opensearch.

I can split this if statement out into two, but I don't see a point to do that since both if statements will have the same body.

Reply:
How? The secret name comes from ES, what will happen if the user just configured OS?

```
{{- if or .Values.global.elasticsearch.tls.existingSecret .Values.global.opensearch.tls.existingSecret }}
- name: keystore
  secret:
    secretName: {{ .Values.global.elasticsearch.tls.existingSecret }}
    optional: false
{{- end }}
```

Reply:
You're right. I'll split them out.

Reply:
The splitting of the if statement is only needed for the volume section. the rest of the if statements related to existing tls secret can remain the same since it's not referencing the opensearch value.